README.md

```diff
@@ -359,7 +359,7 @@ Alternatively, for Ubuntu 18.04 you can use:
 
 ```bash
 # cd to project folder
-cd azure_sdk_for_c
+cd azure-sdk-for-c

```

Comment:
Thanks for fixing this. Can you also fix up the other occurrence of this, in this doc for Windows and Mac:
https://github.com/Azure/azure-sdk-for-c/tree/a25ddc7e2ea821bf8b09c6ed2cfa59ce51e3cf81#windows
https://github.com/Azure/azure-sdk-for-c/tree/a25ddc7e2ea821bf8b09c6ed2cfa59ce51e3cf81#build-1

Reply:
Okay.
Modified.